nautobot_golden_config/nornir_plays/config_intended.py

```diff
@@ -32,7 +35,22 @@
 InventoryPluginRegister.register("nautobot-inventory", NautobotORMInventory)
 LOGGER = logging.getLogger(__name__)
 
-jinja_env = engines["jinja"].env
+# Use a custom Jinja2 environment instead of Django's to avoid HTML escaping
+# Trim_blocks option defaulted to True to match nornir's default environment
+OPTION_LSTRIP_BLOCKS = False
+OPTION_TRIM_BLOCKS = True
```

Comment:
Just showing this for future reference https://github.com/nornir-automation/nornir_jinja2/blob/7ccea50e5b4e5d094047248a586f6c76b1e3f7be/nornir_jinja2/plugins/tasks/template_file.py#L39